client/src/components/game/star/StarDetail.vue

```diff
@@ -28,14 +28,18 @@
           <p class="mb-0" v-if="star.isAsteroidField">This star is surrounded by an <span class="text-warning">Asteroid Field</span>.</p>
           <p class="mb-2 text-info" v-if="star.isAsteroidField"><small><i>Asteroid Fields start with additional natural resources and x2 Defender Bonus (net +2 Weapons).</i></small></p>
         </div>
-        
+
         <div v-if="star.wormHoleToStarId">
           <hr/>
           <p class="mb-0" v-if="wormHolePairStar">This star is a <strong>Worm Hole</strong> to <a href="javascript:;" @click="viewOnMap(wormHolePairStar)"><i class="fas fa-eye mr-1"></i>{{wormHolePairStar.name}}</a>.</p>
           <p class="mb-0" v-if="!wormHolePairStar">This star is a <strong>Worm Hole</strong> to an unknown star.</p>
           <p class="mb-2 text-info"><small><i>Travel between Worm Holes takes 1 tick.</i></small></p>
         </div>
-
+        <div v-if="star.warpGate">
+        <hr/>
+          <p class="mb-0">This star has a <strong>Warp Gate</strong><i class="fas fa-dungeon ml-2"></i>.</p>
```

Comment:
There's also icons for the other star types, it would be nice to have icons displayed on those in the same way as you have done here if it isn't too much effort.

See the compact UI for the icons.